src/promptflow-devkit/promptflow/_sdk/_pf_evals_dependencies.py

```diff
@@ -0,0 +1,16 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# This file captures promptflow-evals dependencies on private API of promptflow.
+# In case changes are made please reach out to promptflow-evals team to update the dependencies.
+
+
+def _get_pf_evals_dependencies():
```

Comment:
why do we need a function here? why not directly have some imports here?

Reply:
Using this method in test to make sure imports are not broken. If file is not loaded then imports command might not run. Open to suggestions if this approach is not full proof.

Reply:
Think clement means you can create a folder which has `__init__.py` inside and import all private api there, and whenever you need these private apis, just import from this init file. Here is a practice you can check out:
[imports_from_azure_ai_ml.md - Repos](https://msdata.visualstudio.com/Vienna/_git/sdk-cli-v2?path=%2Fdocs%2Fmldesigner%2Fimports_from_azure_ai_ml.md&_a=preview)

Reply:
Got pretty close to it. Have imported dependencies in file and then using them in test.